xml/System.Windows.Forms/CloseReason.xml

```diff
@@ -149,7 +149,8 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>The Microsoft Windows Task Manager is closing the application.</summary>
+        <summary><para>The Microsoft Windows Task Manager is closing the application.</para>
+        <para>In Windows 8.1 and later versions, this member is not used, because Task Manager issues the `WM_SYSCOMMAND` message with `SC_CLOSE`. In Windows 7 and earlier versions, the `WM_CLOSE` message was issued. This action is now indistinguishable from and misclassified as  `CloseReason.UserClosing`.</para></summary>
```

Comment:
Oops, forgot this isn't Markdown so can't use backticks.

```suggestion
        <para>In Windows 8.1 and later versions, this member is not used, because Task Manager issues the <c>WM_SYSCOMMAND</c> message with <c>SC_CLOSE</c>. In Windows 7 and earlier versions, the <c>WM_CLOSE</c> message was issued. This action is now indistinguishable from and misclassified as  <c>CloseReason.UserClosing</c>.</para></summary>
```